test-case/check-keyword-detection.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+set -e
+
+##
+## Case Name: check_keyword_detection
```

Comment:
make this the same as file name: check-keyword-detection

Reply:
Fixed.